wails-frontend/src/pages/PasswordPromptHandler/components/clipBoardCopy.tsx

```diff
@@ -0,0 +1,14 @@
+import { ClipboardSetText } from '@wailsjs/runtime';
+import { FiCopy } from 'react-icons/fi';
+
+interface CopyProps {
+  data: string;
+}
+
+export function CopyClip({ data }: CopyProps) {
+  async function handleCopy() {
+    await ClipboardSetText(data);
+  }
+
+  return <FiCopy onClick={handleCopy} style={{ cursor: 'pointer' }} />;
+}
```

Comment:
Why not use the ui kit one instead ? 🤔

Reply:
altough the UI kit one seems a bit more evolued, the main blocker is the the clipboard copy function used here is the wails one, not the browser one.
It could be done in a follow up issue